distribution.md

```diff
@@ -104,3 +105,43 @@ The following labels MUST be set in the buildpack image(through the image config
 The buildpack ID and version MUST match a buildpack provided by a layer blob.
 
 For a buildpackage to be valid, each `buildpack.toml` describing a buildpack implementation MUST have all listed targets.
+
+### Run Image
+
+The following defines how a run image SHOULD be packaged for distribution as an OCI Image. The run image is the component that provides the base from which app images are built.
+
+The image configuration refers to the OCI Image configuration as mentioned [here](https://github.com/opencontainers/image-spec/blob/main/config.md#properties).
+
+#### Image Configuration
+
+The Run Image MUST contain the following configurations:
+
+* Image Config's `config.User` field MUST be set to a non-root user with a writable home directory.
+* Image Config's `os` field MUST be set to the underlying operating system used by the run image.
+* Image Config's `architecture` field MUST be set to the underlying operating system used by the run image.
+
+The Run Image SHOULD contain the following configurations:
+
+* Image Config's `variant` field SHOULD be set to the underlying architecture variant.
+
+#### Environment Variables
+
+The Run Image MUST contain the following Environment Variables:
+
+* Image Config's `config.Env` field MUST have the environment variable `CNB_USER_ID` set to the user UID/SID of the user specified in the `User` field.
+* Image Config's `config.Env` field MUST have the environment variable `CNB_GROUP_ID` set to the primary group GID/SID of the user specified in the `User` field.
+* Image Config's `config.Env` field MUST have the environment variable `PATH` set to a valid set of paths or explicitly set to empty (`PATH=`).
+
+#### Labels
+
+The Run Image SHOULD contain the following Labels on the image configuration:
+
+| Label             | Description
+| --------          | --------
+| `io.buildpacks.distribution.name` |  A string denoting the Operating System distribution
+| `io.buildpacks.distribution.version` | A string denoting the Operating System distribution version
+| `io.buildpacks.id` | <Target ID>
+
+Where,
+
+`<Target ID>` is an identifier specified on the runtime base image that must be provided to buildpacks as `CNB_TARGET_ID` during the build process.
```

Comment:
The env var `CNB_TARGET_ID` is part of the buildpack API. I would not mention it here.

What if we change this sentence to explain what it's intended to be used for instead of how it's provided to the buildpack.

Here's my swing at it:

```suggestion
`<Target ID>` is an identifier specified on the runtime image that MAY be used to apply target-specific logic.
```

I explicitly used **MAY** here because it's irrelevant to **this spec** whether the ID is used at all. I envision that the _**MUST**_ requirement would be part of the buildpack API.

@natalieparellano / @samj1912 any contradictory opinions or alternative suggestions?